packages/backend/test/core/reports/createReports.test.ts

```diff
@@ -35,3 +38,87 @@ describe(createReports.name, () => {
     ])
   })
 })
+
+describe(aggregateBalancesPerProject.name, () => {
+  it('discards balances before bridge sinceTimestamp', () => {
```

Comment:
```suggestion
  it('skips balances before bridge sinceTimestamp', () => {
```

Reply:
Do I understand it correctly that there is no discarding, just filtering ?

Reply:
Yes, that's right.